### ISSUE TITLE
Provide meaningful defaults in the `values.yaml`

### ISSUE THREAD

**Issue by fengelniederhammer** [loculus-project/loculus#3070]

**Comment by theosanderson** [comment]
I agree with the general theme of this, but have comments on a couple of the points.

> The organisms. It doesn't make sense that the Helm chart comes with organisms configured.

That's is a matter of opinion I would say. The current set up is intentional whereby if you don't specify any organisms you get some default ones is intentional. (It might make sense to simplify these defaults rather than having so many fields as at present). The presence of these fallback defaults allows a simple install to try out the Helm chart without having to think about organism configuration. It seems reasonable to me. 

> dataUseTermsUrls:

Some day we should make the whole concept of these terms disableable. But until then we will likely need some default that users replace (but could remove `MVP`)

**Comment by chaoran-chen** [comment]
Another one:

S3 is currently enabled by default, but we probably don't want that. Changing it might, however, depend on moving the default organisms because we have "Test organism (with files)" as a default so that it is included in the previews and E2E tests which needs S3.
